src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -38,6 +39,25 @@ import { MatSelectSearchClearDirective } from './mat-select-search-clear.directi
 /** The max height of the select's overlay panel. */
 const SELECT_PANEL_MAX_HEIGHT = 256;
 
+/** InjectionToken that can be used to specify global options. */
+export const MATSELECTSEARCH_GLOBAL_OPTIONS = new InjectionToken<MatSelectSearchOptions>('MATSELECTSEARCH_GLOBAL_OPTIONS');
+
+/** Configurable options for MatSelectSearch. */
```

Comment:
Maybe we could use something like

type MatSelectSearchOptions = Pick<MatSelectSearchComponent, 'input1' | 'input2'> & Partial<MatSelectSearchComponent>

See https://stackoverflow.com/a/57070274/7596814

Reply:
(This is optional, but would help stricter type checks, and ut ensures the properties actually exist)

Reply:
Like! Comes soon.